### PR TITLE
chore: Update CI and examples for Terraform

### PR DESCRIPTION
**Pull request by npalm**
- upgrade terraform versions
- replaced philips vpc module by aws-terraform-vpc module
- dropped support for terrform pre 1.x from examples

### CODE REVIEW

File: .github/workflows/terraform.yml [hunk 1]
```diff
@@ -31,7 +29,8 @@ jobs:
           touch modules/runner-binaries-syncer/lambdas/runner-binaries-syncer/runner-binaries-syncer.zip
       - name: terraform init
         run: terraform init -get -backend=false -input=false
-      - name: check terraform formatting
+      - if: contains(matrix.terraform, '1.1.')
```
Comment (ScottGuymer) [inline .github/workflows/terraform.yml:32]: Why not run this on `latest`?

**Review by ScottGuymer — APPROVED** [review]

**Comment by ScottGuymer** [comment]
There is some conflict here. Lets solve it and get this merged.

**Comment by npalm** [comment]
Will do asap.

File: .github/workflows/terraform.yml (hunk 1 above)
Comment (npalm) [reply .github/workflows/terraform.yml:32]: Added an extra check for the latest, but allowing errors to keep the build immutable.

**Review by ScottGuymer — DISMISSED** [review]
Apart from formatting it looks good.

File: .github/workflows/terraform.yml [hunk 2]
```diff
@@ -54,9 +56,5 @@ jobs:
       - uses: actions/checkout@v2
       - name: terraform init
         run: terraform init -get -backend=false -input=false
-      - if: contains(matrix.terraform, '1.0.')
-        name: check terraform formatting
```
Comment (ScottGuymer) [inline .github/workflows/terraform.yml:58]: Why remove the formatting check?

Comment (npalm) [reply .github/workflows/terraform.yml:58]: Reverted. I had the impression that it is fully recursive, thought I was looking on the the module job that was fialing but it was an example

**Review by ScottGuymer — APPROVED** [review]

**Review by ScottGuymer — APPROVED** [review]

**Pull request merged**